examples/testnet.yaml

```diff
@@ -7,5 +7,7 @@ testnet-croeseid-1:
       # consensus_key:
       #   priv:
       #   pub:
+      # optional, using existing pubkey instead of using priv_validator_key.json, generated by tmkms
+      # pubkey: tcrocnclconspub1.....
```

Comment:
It already supports setting "consensus_key", the "priv_validator_key.json" will be overwritten by that. But you need to put private key in that.